src/js/main.js

```diff
@@ -537,7 +537,7 @@ function startProcess() {
                 analytics.setDimension(analytics.DIMENSIONS.CONFIGURATOR_EXPERT_MODE, checked ? 'On' : 'Off');
             });
 
-            if (FEATURE_CONFIG) {
+            if (FEATURE_CONFIG && FEATURE_CONFIG.features !== 0) {
```

Comment:
lol, considering that `FEATURE_CONFIG.features` is meant to be an instance of `Feature`, the default of `0` is confusing.
I think we should change the default to `undefined` and then check for that.